CHANGELOG.md

```diff
@@ -1,317 +1,853 @@
+* Rename `appromixate-size-test.js` to `approximate-size-test.js` (@ralphtheninja)
 
-## 0.4.0 @ Dec 18 2013
+### Removed
+* Remove non supported versions from Travis (@ralphtheninja)
 
-  * remove old, unused util functions @rvagg
-  * speed up batch() & allow non-Strings to C++ @rvagg
-  * fix batch() benchmarks @rvagg
-  * improved compression test @rvagg
-  * added SQLite3 to test suite @rvagg
-  * remove compile warnings on osx @rvagg
-  * return Strings not Buffers from C++ when possible @rvagg
-  * optimised encoders & decoders @rvagg
-  * added basic get() benchmarks @rvagg
-  * revamped benchmark suite @rvagg
-  * allow JS Strings through to native layer @rvagg
-  * cleaner build for osx @rvagg
-  * remove compile warnings for solaris @rvagg
-  * LevelDB 1.7 @rvagg
-  * added `compress` boolean on open() @rvagg
+### Fixed
+* Ensure Travis can compile in case no prebuilt binaries can be found (@ralphtheninja)
+* Fix deprecation test (@juliangruber)
 
-## 0.3.x and prior
+## [1.3.1] - 2015-12-10
 
-  * stuff
+### Added
+* Add node 5 to travis (@ralphtheninja)
+
+### Changed
+* Update outdated dependencies (@ralphtheninja)
+* Test on latest node 2, node 3 (@ralphtheninja)
+
+## [1.3.0] - 2015-11-12
+
+### Changed
+* README: fixed small typo (Stephen Sawchuck)
+* README: update url to Snappy (@hansott)
+* README: add dependency badge (@ralphtheninja)
+* Test on all major abi versions (@ralphtheninja)
+* Update outdated dependencies (@ralphtheninja)
+* Track and expose chained batch ops queue length (@kemitchell)
+
+### Fixed
+* Dev depend on `tap` to fix `npm@3` warning (@ralphtheninja)
+
+## [1.2.1] - 2015-06-10
+
+### Changed
+* Improve error message when trying to require `leveldown` (@ralphtheninja)
+
+## [1.2.0] - 2015-06-04
+
+### Changed
+* Less restrictive version on `leveldown` (@ralphtheninja)
+
+### Fixed
+* Handle errors in benchmarks (@juliangruber)
+
+## [1.1.1] - 2015-05-29
+
+### Added
+* Add link to `level/community` (@ralphtheninja)
+
+### Changed
+* Update `leveldown` dependency (@juliangruber)
+
+### Removed
+* Remove compression tests (@juliangruber)
+
+## [1.1.0] - 2015-05-17
+
+### Changed
+* Batch operation default to `'put'` (@ralphtheninja)
+
+## [1.0.0] - 2015-05-14
+
+### Removed
+* Remove return values from `dispatchError()` and `readError()`, they are used as voids (@ralphtheninja)
+
+## [1.0.0-5] - 2015-05-07
+
+### Changed
+* Target multiple iojs versions, remove notifications (@ralphtheninja)
+* Deprecate static functions `destroy()` and `repair()` (@juliangruber)
+
+## [1.0.0-4] - 2015-05-06
+
+### Changed
+* Deprecate `.approximateSize()` (@juliangruber)
+
+## [1.0.0-3] - 2015-05-05
+
+### Changed
+* Replace `tap` with `tape` + `faucet` (@ralphtheninja)
+* Refactor read streams using `level-iterator-stream` and `level-codec` (@juliangruber)
+
+## [1.0.0-2] - 2015-04-30
+
+### Changed
+* Refactor ltgt encoding (@juliangruber)
+
+### Fixed
+* Fix readStream *AsBuffer options (@juliangruber)
```

Comment:
Escape the asterisk

Reply:
Aaaah, this is why the diff looked weird in github.